test/cli/package-test-simple/main_lib/__package.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
```

Comment:
Why are all of these defined as `cli` tests rather than `testdata` tests?

`testdata` tests are tested in LSP and CLI modes, whereas these are only tested in whatever command you run.

Reply:
The cli test is only here for the basics, the more extensive tests are in testdata. That includes missing test imports, test imports in non-test code etc.